### PR TITLE
Bump astroid to 2.11.7, update changelog

### PR DESCRIPTION
**Pull request by DanielNoord**
## Steps

- [x] Write a good description on what the PR does.

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :hammer: Refactoring   |

## Related Issue

### CODE REVIEW

**Comment by DanielNoord** [comment]
Let's not worry about `pre-commit`. We're closing this branch immediately afterwards anyways.

File: CONTRIBUTORS.txt [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# This file is autocompleted by 'contributors-txt',
+# using the configuration in '/Users/daniel/DocumentenLaptop/Programming/Github/astroid/script/.contributors_aliases.json'.
+# Do not add new persons manually and only add information without
+# using '-' as the line first character.
+# Please verify that your change are stable if you modify manually.
+
```
Comment (Pierre-Sassoulas) [inline CONTRIBUTORS.txt:6]:
```suggestion
```
Apparently the header only 'worked on my machine' :smile:

**Review by Pierre-Sassoulas — CHANGES_REQUESTED** [review]
Could you rebase interactively the suggestion I made so the historic is clean ? The pre-commit issue if the same than what we had the last 6 patch release, LGTM.

File: CONTRIBUTORS.txt (hunk 1 above)
Comment (DanielNoord) [reply CONTRIBUTORS.txt:6]: What's bad about this warning?

Comment (Pierre-Sassoulas) [reply CONTRIBUTORS.txt:6]: It's duplicated it should not have been added by contributor-txt. (see 4 lines under)

**Pull request merged**